icd/vksc_device.cpp

```diff
@@ -10,6 +10,7 @@
 #include "vksc_queue.h"
 #include "vksc_command_buffer.h"
 #include "vksc_instance.h"
+#include "vksc_global.h"
```

Comment:
Why has global come in as an include when that file hasn't been touched? If something was out of scope that global transitively pulls in, perhaps we should include that here. Or is this a leftover from development?